src/vdaf/prio3_test.rs

```diff
@@ -149,6 +149,19 @@ fn test_vec_prio3_sum() {
     }
 }
 
+#[test]
```

Comment:
This test should be removed. It would only be appropriate if we had test vectors `Prio3Aes128Average`, but we don't (yet!).